x-pack/plugins/siem/public/components/error_toast/index.tsx

```diff
@@ -22,40 +21,33 @@ interface ReduxProps {
 }
 
 interface DispatchProps {
-  addError?: ActionCreator<{ id: string; title: string; message: string }>;
-  removeError?: ActionCreator<{ id: string }>;
+  removeError: ActionCreator<{ id: string }>;
 }
 
 type Props = OwnProps & ReduxProps & DispatchProps;
 
-const ErrorToastComponent = pure<Props>(({ toastLifeTimeMs = 10000, errors = [], removeError }) =>
-  globalListFromToasts(errorsToToasts(errors), removeError!, toastLifeTimeMs)
-);
-
-export const globalListFromToasts = (
-  toasts: Toast[],
-  removeError: ActionCreator<{ id: string }>,
-  toastLifeTimeMs: number
-) =>
-  toasts.length !== 0 ? (
-    <EuiGlobalToastList
-      toasts={toasts}
-      dismissToast={({ id }) => removeError({ id })}
-      toastLifeTimeMs={toastLifeTimeMs}
-    />
-  ) : null;
-
-export const errorsToToasts = (errors: appModel.Error[]): Toast[] =>
-  errors.map(({ id, title, message }) => {
-    const toast: Toast = {
-      id,
-      title,
-      color: 'danger',
-      iconType: 'alert',
-      text: <p>{message}</p>,
-    };
-    return toast;
+const ErrorToastComponent = ({ toastLifeTimeMs = 5000, errors = [], removeError }: Props) => {
```

Comment:
Why is the default lifetime changed from 10 seconds to 5 seconds?

Doesn't the guidelines state:

```ts
By default, a toast stays on the screen 10 seconds. Users should be able read the message in 6 to 7 seconds. The message should get straight to the point and rarely include more than one line.
```

Or am I missing somewhere in the guidelines that toasts should be visible for 5 seconds?

Reply:
I did put 5s here because you do not have too much to read anymore, you just need to click on the button to see the errors